### DISCUSSION TITLE
Add a data attribute to hide html elements in layer panel.

### DISCUSSION THREAD

**Discussion by ronaldohoch** [GrapesJS/grapesjs#4368] [Ideas]
# Context 
I have a configuration element in my mjml plugin, it uses `mj-html-attributes` using this structure:
```js
  <div id="gjs" style="height:0px; overflow:hidden;">
    <mjml>
      <mj-head id="mjmlHead">
        <mj-html-attributes style="display:none;" id="mjmlAttributesRss">
          <mj-selector path=".dinRss" id="mjmlAttributesRssWrapper">
            <mj-html-attribute id="attribute-rss-url" name="rss-url">https://feed.somewebsite.com/rss</mj-html-attribute>
          </mj-selector>
        </mj-html-attributes>
      </mj-head>
      <mj-body></mj-body>
    </mjml>
  </div>
```

It is set as display:none, so it didn't appear in editor. But, in the layer tab, it is shown and user can click and delete the element.

![image](https://user-images.githubusercontent.com/2287371/172612872-70aeef25-d2a7-4997-8af4-e03864244d04.png)

# Idea

Create a data-gjs-something to hide the element inside the layer tab.

**Comment by artf** [comment]
Hi Ronaldo, did you try with `data-gjs-layerable="false"`?

**Reply by ronaldohoch** [reply]
Didn't know it! Worked here.

Thank you! :)
